canonical-web-and-design/canonical.com
# Build the homepage content with suru strips

## Done
Updated some base template content like adding favicons
Built the base page with content from the design
Wrote a number of suru strip styles

## QA
- Go to http://0.0.0.0:8002/
- Check it matches [the design](https://drive.google.com/file/d/114QpYLu8SXvqd_5f8caFOvsrC0TTuSSW/view)

## Screenshot
![Screenshot_2019-07-18 Canonical The company behind Ubuntu](https://user-images.githubusercontent.com/1413534/61494834-3419fc80-a9af-11e9-8cde-29f05fa4952b.jpg)


Fixes https://github.com/canonical-web-and-design/web-squad/issues/1371
